### ISSUE TITLE
Make file associations directly in the program

### ISSUE THREAD

**Issue by rellikmil** [bluescan/tacentview#185]
Is it possible to make file associations directly in the program, similar to video players and other image viewers?

I apologize if such an issue has already been created, and thank you very much for Tacent View, it may not open everything I need, but it does it super fast and without lagging.

**Comment by bluescan** [comment]
I assume Windows? I do sorta wonder if that's best left to the OS and just choosing the app you want to open a particular extension... especially since there's no real installer for windows (it only comes as a portable zip). This could end up with the registry being out of wack if a new portable release is installed to a different directory.

**Comment by rellikmil** [comment]
Yes, Windows. If you leave it to the OS, you will get tired of selecting each format yourself, I would like to see something like in 7-Zip and Media Player Classic to make associations with files with one button
